FoodOrderingApp-service/src/main/java/com/upgrad/FoodOrderingApp/service/entity/OrderEntity.java

```diff
@@ -0,0 +1,149 @@
+package com.upgrad.FoodOrderingApp.service.entity;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+@NamedQueries({
+        @NamedQuery(name = "fetchAllOrders", query = "select o from OrderEntity o where o.customerEntity=:custEntity"),
+        @NamedQuery(name = "fetchOrderByAddress", query = "select o from OrderEntity o where o.addressEntity=:addrEntity"),
+        @NamedQuery(name = "getOrdersByRestaurantId", query = "select o from OrderEntity o where  o.restaurantEntity=:restEntity")
+})
+
+@Entity
+@Table(name = "orders")
+public class OrderEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID")
+    private int id;
+
+    @Column(name = "UUID")
+    @Size(max = 200)
+    private String uuid;
+
+    @Column(name = "BILL")
+    private BigDecimal bill;
+
+    @Column(name = "DISCOUNT")
+    private BigDecimal discount;
+
+    @Column(name = "DATE")
+    private ZonedDateTime date;
+
+//    @ManyToOne
+//    @JoinColumn(name = "COUPON_ID")
+//    private CouponEntity couponEntity;
+
+    @OneToOne
+    @JoinColumn(name = "PAYMENT_ID")
+    private PaymentEntity paymentEntity;
+
+    @ManyToOne
+    @JoinColumn(name = "CUSTOMER_ID")
```

Comment:
This customer_id entity is throwing run time error. Can you please check ?

2020-09-26 19:13:14.585 ERROR 9050 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: com.upgrad.FoodOrderingApp.service.entity.CustomerEntity.address in com.upgrad.FoodOrderingApp.service.entity.AddressEntity.customer